src/main.ts

```diff
@@ -1,74 +1,104 @@
-import { app, BrowserWindow, Menu, ipcMain } from 'electron';
+import { spawn, ChildProcess } from 'child_process';
+import { app, BrowserWindow, ipcMain } from 'electron';
+import { createTopMenu } from './main/menu';
+import { pathToFileURL } from 'url';
+import { configMaestro } from './utils/config';
+import { setSessionId, getSessionId } from './cosimulation/simulationContext';
+
 import * as path from 'path';
+import * as fs from 'fs';
+import { exec } from 'node:child_process';
+import kill from 'tree-kill';
+
 let mainWindow: BrowserWindow | null = null;
+let maestroProcess: ChildProcess | null = null;
+const { coeJarPath, simulationConfigPath, fmusPath, multiModels, outputPath } = configMaestro;
+
+if (!coeJarPath || !simulationConfigPath || !fmusPath || !multiModels || !outputPath) {
+  console.error('Missing required paths in config.json');
+  process.exit(1);
+}
+
+const isDev = process.env.NODE_ENV === 'development';
+process.env.NODE_ENV =
+  process.env.NODE_ENV || (isDev ? 'development' : 'production');
+
+const preloadPath = isDev
+  ? path.resolve(__dirname, 'preload.js')
+  : path.resolve(app.getAppPath(), 'dist/preload.js');
+
+const startUrl = isDev
+  ? 'http://localhost:3000'
+  : `file://${path.resolve(app.getAppPath(), 'dist/index.html')}`;
+
+const iconPath = isDev
+  ? path.resolve(__dirname, 'resources/into-cps/appicon/into-cps-logo.png.ico')
+  : path.resolve(
+      app.getAppPath(),
+      'dist/resources/into-cps/appicon/into-cps-logo.png.ico',
+    );
+
+const maestroJarPath = isDev
+  ? path.resolve(__dirname, 'resources/maestro/maestro.jar')
+  : path.resolve(app.getAppPath(), 'dist/resources/maestro/maestro.jar');
```

Comment:
Name of the maestro file can come as constant from `config.ts`

Reply:
Name of the maestro file has been declared in `config.ts`.